frontend/src/components/OdhDocCard.tsx

```diff
@@ -134,8 +134,6 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFavorit
     <Card
       data-id={odhDoc.metadata.name}
       id={odhDoc.metadata.name}
-      role="article"
-      isHoverable
```

Comment:
I presume we are removing this prop since it's deprecated right?
Dont know why we kept it since we already had isSelectable.

Reply:
Yes that's correct @lucferbux !